docs/source/intro/get_started.rst

```diff
@@ -0,0 +1,123 @@
+.. _intro/get_started:
+
+****************
+Getting Started
+****************
+
+Installation
+============
+
+There are numerous routes to setting up the full AiiDA environment, depending on your use case.
+Here we first provide the simplest approach for installation on your local computer.
+
+`Conda <https://docs.conda.io>`_  provides a cross-platform package management system, from which we can install the basic components of the AiiDA infrastructure in an isolated environment:
+
+.. code-block:: console
+
+    $ conda create -n aiida -c conda-forge aiida-core aiida-core.services
+    $ conda activate aiida
+    $ reentry scan
```

Comment:
Could this be rolled into `conda activate aiida`?

Reply:
Possibly. Me and @ltalirz have already discussed this for example in https://github.com/chrisjsewell/activate_aiida/issues/4, and I think he is looking into it